rsp/00x_bz_rest_api.md

```diff
@@ -0,0 +1,159 @@
+# RSP - 1 — *bz* REST API
+
+## `POST /login`
+
+### Synopsis
+
+Use this method to authenticate.
+
+### Request body
+
+```json
+{
+    "email": "$(email: String)",
+    "passwd": "$(passwd: String)"
+}
+```
+
+### Response body
+
+```json
+{
+    "id": "$(id: Int)",
+    "infos": {
+        "email": "$(email: String)"
+    }
+}
+```
+
+### Response status
+
+Code | Description
+-----|-------------
+200  | Success
+400  | Error in request body
+403  | This account is already connected
+404  | No account for the given url
+
+## GET `/:id/characters`
```

Comment:
Could we prefix this with something, like `/user/:id/characters`? This would avoid any problems if we change the format of `:id`.